pkg/postage/batchstore/store_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package batchstore_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/postage/batchstore"
+	"github.com/ethersphere/bee/pkg/statestore/mock"
+)
+
+func TestStore(t *testing.T) {
+
+}
+
+func TestStoreMarshalling(t *testing.T) {
+	mockstore := mock.NewStateStore()
+	store, err := batchstore.New(mockstore)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	newPrice := big.NewInt(99)
```

Comment:
Perhaps a line of comment as to what we are trying to achieve here? As in: creating the store, changing the price, expecting different result.